conference/DS/isca.yml

```diff
@@ -0,0 +1,15 @@
+- title: ISCA
+  description: International Symposium on Computer Architecture
+  sub: DS
+  rank: A
+  dblp: isca
+  confs:
+    - year: 2022
+      id: isca22
+      link: https://www.iscaconf.org/isca2022/
+      timeline:
+        - abstract deadline: '2021-11-16 23:59:59'
+        - full paper deadline: '2021-11-23 23:59:59'
```

Comment:
Hi, we *do not* provide these field names, please refer to the following two examples:
[1. using multiple deadlines](https://github.com/ccfddl/ccf-deadlines/pull/48/files)
[2. using hybrid deadlines](https://github.com/ccfddl/ccf-deadlines/blob/main/conference/DB/icdt.yml)